core/src/test/java/hudson/util/ConsistentHashTest.java

```diff
@@ -183,18 +185,20 @@ public String hash(String str) {
      * This test doesn't fail but it's written to measure the performance of the consistent hash function with large data set.
      */
     @Test
+    @Ignore("Helper test for performance, no assertion")
```

Comment:
Fine with me, is the performance difference for this implementation noteworthy?

Reply:
No tested due to [Hash function comparison](https://automationrhapsody.com/md5-sha-1-sha-256-sha-512-speed-performance/#:~:text=SHA%2D1%20is%20fastest%20hashing,and%2023.4%25%20for%20longer%20strings.), the difference should be around 5-10% between MD5 and SHA-256.

It's not like we are moving from MD5 to BCrypt :)